docs/technical/cdl.md

```diff
@@ -0,0 +1,28 @@
+# Controlled Digital Lending (CDL)
+
+With Controlled Digital Lending we allow digital access to a resource while ensuring that access does not exceed the access that would be permitted to the physical item. To achieve this, an item selected for CDL is sequestered physically, then only a single user can access it digitally at any given time. If two copies of an item are sequestered, then two users can access the digital copy, et cetera.
+
+In Figgy the CDL access business logic can be found in `app/services/cdl/` but its orchestration ranges through the uv_manager javascript, the viewer controller, and the cdl controller. Of course authorization logic can also be found in `ability.rb`. Bibdata provides access to data needed from Alma.
+
+What follows is a fairly detailed step-through of CDL access logic.
+
+- when figgy javascript goes to load the viewer, it submits a HEAD request to the manifest. If a 401 unauthorized is received, figgy redirects to /viewer/auth.
```

Comment:
Is this still true after  our graphql refactor?

Reply:
nope!